src/mocks/keyring.mock.ts

```diff
@@ -1,23 +1,23 @@
 class KeyringMockWithInit {
```

Comment:
Is there a way we can ensure this conforms to the keyring interface? Can we say something like this?

``` suggestion
class KeyringMockWithInit implements KeyringClass {
```